arangod/Aql/AqlCallStack.cpp

```diff
@@ -69,6 +69,16 @@ auto AqlCallStack::popCall() -> AqlCallList {
   return call;
 }
 
+void AqlCallStack::popDepthsLowerThan(size_t depth) {
+  TRI_ASSERT(!_operations.empty());
+  for (auto i = _operations.size() - depth; i < _operations.size(); ++i) {
+    auto& operation = _operations[i];
+    if (operation.hasMoreCalls()) {
+      std::ignore = operation.popNextCall();
+    }
```

Comment:
Can an operation in theory have more than one call? If so, do we need to repeat the `if (hasMoreCalls()) pop()` logic here until there are no more calls left?